### ISSUE TITLE
train_CAN 

### ISSUE THREAD

**Issue by xuyingtong1** [mingcv/Bread#5]
I am sorry  to bother you,I'm rather confused about this line of code
`colors = self.model_canet(torch.cat([texture_in, cb_in, cr_in, texture_gt, cb_en, cr_en], dim=1))`
I didn't find the formula and related explanation for this line of code given in the article. I would like to ask what problem and purpose this line of code is going to solve? Looking forward to your reply！

**Comment by mingcv** [comment]
Please take a look at our IJCV version of the paper.
![image](https://github.com/mingcv/Bread/assets/31566437/d0a7f6af-5117-4c43-8e32-5a07a942c611)
Moreover, MECAN is recommended over the CAN model. The above formulation corresponds to the training procedure of MECAN.

https://github.com/mingcv/Bread/blob/e0c50680f38ac9f417ef6a233448f94e68f332f2/train_MECAN.py#L77C18-L77C18

**Comment by mingcv** [comment]
If you have any further questions, don't hesitate to contact me.
